scripts/scriptassist.pl

```diff
@@ -204,7 +204,13 @@ sub get_names {
     $xname =~ s/\W/_/g;
     my $pname = "${xname}::";
     if ($xname ne $sname || $sname =~ /_/) {
-	my $dir = Irssi::get_irssi_dir()."/scripts/";
+	my $dir = "";
+	if (Irssi::is_xdg_supported()) {
+	    $dir = Irssi::get_runtime_dir()."/scripts/";
```

Comment:
here it should also be the data_dir, no?

Reply:
Yep, I missed it, my bad